README.md

```diff
@@ -0,0 +1,38 @@
+## Description
+
+Build two websites for two local organizations: Funny Bone and Ethan Stark.  Deploy them.
+
+## Specifications
+
+### Funny Bone
+- [ ] Implement a e-commerce application
+  - [ ] Administrator view is separate from visitors view
+  - [ ] Implement OAUTH using GMAIL for administrator access (as an administrator, i can log into the website)
+  - [ ] As an administrator, I can add inventory
+  - [ ] As an administrator, I can delete inventory
+  - [ ] As an administrator, I can update inventory
+  - [ ] As a an administrator, I can search for inventory
+  - [ ] As a visitor, I can view store's inventory
+  - [ ] As a visitor, I can send administrator inquiries about specific inventory
+  - [ ] As a visitor, I can browse inventory by category
+- [ ] Deploy to Heroku
+
+### Ethan Stark
+- [ ] Build a personal portfolio site showcasing web development projects
+- [ ] As a visitor, I can easily contact Ethan Stark via email or phone
+- [ ] As a visitor, I can navigate to related links: Github, project sites, LinkedIn
+- [ ] As a visitor, I can browse a gallery of web projects
+- [ ] As a visitor, I can click a web project and view additional details
+- [ ] Site is deployed to EthanStark.com
+
+
+## Quality Rubric
```

Comment:
I would add points to these items. Weight the items highly that you think will contribute most to your learning.